### PR TITLE
TC-804 dropdown height

### PR DESCRIPTION
**Pull request by kscoulter**
Change dropdown height

### CODE REVIEW

File: src/client/components/atoms/datalist-dropdown/datalist-dropdown.jsx [hunk 1]
```diff
@@ -13,10 +13,10 @@ const DatalistDropdwon = props => {
   )
 }
 
-DatalistDropdwon.propTypes = {
+DatalistDropdown.propTypes = {
```
Comment (peggles2) [inline src/client/components/atoms/datalist-dropdown/datalist-dropdown.jsx:16]: nice catch

File: src/client/components/atoms/multiselect/multiselect.jsx [hunk 2]
```diff
@@ -15,28 +14,6 @@ class MultiSelect extends React.Component {
...
-      // } else {
-      //   // Fix an issue where the user can type text in the hidden input inside
-      //   // react-select's dropdown when the dropdown is focused.
-      //   const input = combobox.querySelector('input')
-      //   input.setAttribute('aria-readonly', true)
-      //   input.setAttribute('readonly', true)
-      // }
-    }
-  }
-
```
Comment (peggles2) [inline src/client/components/atoms/multiselect/multiselect.jsx:39]: nice cleanup

File: src/client/components/pages/office-lookup-page/office-lookup-page.jsx [hunk 3]
```diff
@@ -104,7 +104,18 @@ class OfficeLookupPage extends React.PureComponent {
...
+      terms: [
+        'SCORE Business Mentoring',
+        'Small Business Development Center',
+        'U.S. Export Assistance Center',
+        'Veteran’s Business Outreach Center',
+        'Women’s Business Center',
+        'Procurement Technical Assistance Center',
+        'Certified Development Company'
+      ]
+    }
```
Comment (peggles2) [inline src/client/components/pages/office-lookup-page/office-lookup-page.jsx:118]: Can you add a test that makes sure that only these terms are being searched and nothing else?

**Review by peggles2 — CHANGES_REQUESTED** [review]

File: src/client/components/pages/office-lookup-page/office-lookup-page.jsx (hunk 3 above)
Comment (kscoulter) [reply src/client/components/pages/office-lookup-page/office-lookup-page.jsx:118]: @peggles2 can you elaborate on that? What part should I test?

Comment (peggles2) [reply src/client/components/pages/office-lookup-page/office-lookup-page.jsx:118]: Looks like this was from the other pull request so I will approve this ticket and you can address it in the other pull request.

**Review by peggles2 — APPROVED** [review]

**Pull request merged**